.golangci.yml

```diff
@@ -0,0 +1,27 @@
+# ---
```

Comment:
Uncommenting these also a good idea 🤔

Reply:
I've addressed the issues identified in this PR in a new pull request. Please review the updates there. Thank you!